v2/robotmk/runner.py

```diff
@@ -3,13 +3,18 @@
 import dataclasses
 import enum
 import pathlib
-import uuid
 from collections.abc import Iterator, Sequence
 from typing import Final
 
 PYTHON_EXECUTABLE: Final = pathlib.Path("python")
 
 
+@dataclasses.dataclass(frozen=True)
+class Identifier:
+    name: str
+    timestamp: str
```

Comment:
OTI: int / str / datetime? so many options

Reply:
Let's go with str for now, since we use this in multiple places. Otherwise, we would have either duplicate code or some sort of helper function to make this a str.